storage/storagedriver/index.md

```diff
@@ -292,8 +292,8 @@ layer. This means that the writable layer is as small as possible.
 
 When an existing file in a container is modified, the storage driver performs a
 copy-on-write operation. The specifics steps involved depend on the specific
-storage driver. For the default `aufs` driver and the `overlay` and `overlay2`
-drivers, the copy-on-write operation follows this rough sequence:
+storage driver. For the `aufs`, `overlay` and `overlay2` drivers, the 
```

Comment:
nit: the documentation uses Oxford comma, so there should be a comma before `and overlay` 😅 

(just a nit, so I'm ok with merging as-is, if it's not updated)